tests/test_workflow_management.py

```diff
@@ -228,7 +228,8 @@ def stepFive(x: int) -> int:
     with SetWorkflowID(fork_id):
         forked_handle = DBOS.fork_workflow(wfid, 3)
     assert forked_handle.workflow_id == fork_id
-    assert forked_handle.get_status().app_version == DBOS.application_version
+    app_version = forked_handle.get_status().app_version
+    assert app_version is None or app_version == DBOS.application_version
```

Comment:
Uh it's not testing the change because this test still passes with the old code.

Reply:
It's this way because dequeuing a workflow sets its version, so depending on timing the version can be one or the other